doc/zypper.8.txt

```diff
@@ -1906,6 +1906,15 @@ This directory is used by all ZYpp-based applications.
 +
 This directory is used by all ZYpp-based applications.
 
+*/var/log/zypper.log*::
+	Zypper log file. It should be attached to any bugreport. (see also zypper-log(8)).
```

Comment:
```suggestion
	Zypper log file. It should be attached to all bugreports. (see also zypper-log(8)).
```
"Any" is iirc more used in the context of "beliebig, irgendein" and not for "jeder/alle"